pkg/client/results/e2e/consts.go

```diff
@@ -19,6 +19,6 @@ package e2e
 
 const (
 	// ResultsDirectory is the directory where the results will be found
-	ResultsSubdirectory = "e2e/results/"
+	ResultsSubdirectory = "e2e/results/global/"
```

Comment:
This fixes the e2e command for now, but unlike the rest of the results stuff, it wasn't versioned.

I could modify this to look in the appropriate location based on the version. One of the frustrating things about this is that the config readers are not rewindable so you only get one read-per open. That would be great to be able to change that.

Reply:
So even I learned something going through this code. I originally thought it was versioned independently of Sonobuoy because I saw things like [this](https://github.com/heptio/sonobuoy/blob/master/pkg/client/results/reader.go#L51-L54). However, the version it uses (to compare those to) is actually the version stored in the config file; i.e. the Sonobuoy version.

Those are just versioning it based on the major/minor number and not patch releases which is why I (in another snippet of code) moved to using go-version which will let me do more complex checks with versions that look like semver (the existing checks in the Reader just want to know if the version has that prefix, e.g. has the same major/minor version)